packages/client/src/pages/RecommendationsPage/RecommendationsTable/RecommendationsTable.tsx

```diff
@@ -62,12 +68,53 @@ const getColumns = (useKilograms: boolean): GridColDef[] => [
   },
 ]
 
+function CustomPagination(
```

Comment:
Since this is a component, I would extract this into its own file within the same folder